randobot/handler.py

```diff
@@ -907,15 +990,38 @@ async def check_seed_status(self):
                 await self.check_seed_status()
         elif status == 1:
             await self.load_seed_hash()
+            await self.load_seed_password()            
         elif status >= 2:
             self.state['seed_id'] = None
             await self.send_message(
                 'Sorry, but it looks like the seed failed to generate. Use '
                 '!seed to try again.'
             )
+            
+
+    async def load_seed_password(self, manual=False):
+        seed_password = self.zsr.get_password(self.state['seed_id'])
+        if seed_password == None:
+            if manual:
+                return False
+            elif self.state['password_retry'] > 2:
+                await self.send_message(
+                    'Sorry, but it looks like the password for this seed cannot be retrieved.'
+                    'Please wait a few minutes and try manually before race start using !password get'
+                )
+            else: 
+                self.state['password_retry'] +=1
+                await sleep(120)
+                await self.load_seed_password()
```

Comment:
This will lock up the bot and prevent it from doing anything for 2 minutes. I'm not sure if there's an easy way to code "do this thing 2 minutes from now" with the way we've set up the event loop. Might be doable but TBH I'd just take out this section and just tell the user to call `!password get`. Hopefully errors like this will not be too common anyway, and `get_password` already does retries as well.